docs/fundamentals/package-validation/baseline-version-validator.md

```diff
@@ -77,3 +73,66 @@ public static HttpClient Connect(string url, TimeSpan timeout)
 Now when you pack the project, it succeeds.
 
 ![BaselineVersionSuccessful](media/baseline-version-successful.png)
+
+For version 2.0.0 you later decide you want to remove obsolete `Connect` method with the single `string` parameter and upon careful consideration deliberately decide to accept this breaking change.
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <PackageVersion>2.0.0</PackageVersion>
+    <EnablePackageValidation>true</EnablePackageValidation>
+    <PackageValidationBaselineVersion>1.1.0</PackageValidationBaselineVersion>
+  </PropertyGroup>
+
+</Project>
+```
+
+```diff
+- public static HttpClient Connect(string url)
+- {
+-     return Connect(url, Timeout.InfiniteTimeSpan);
+- }
+
+public static HttpClient Connect(string url, TimeSpan timeout)
+{
+    // ...
+}
+```
+
+To suppress the `CP0002` error for this intentional breaking change, you can add a `CompatibilitySuppressions.xml` file to your project.
+The easiest way to add it is by adding the `GenerateCompatibilitySuppressionFile` property to your project (see also [Suppress compatibility errors](overview.md#suppress-compatibility-errors)).
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <PackageVersion>2.0.0</PackageVersion>
+    <EnablePackageValidation>true</EnablePackageValidation>
+    <PackageValidationBaselineVersion>1.1.0</PackageValidationBaselineVersion>
+    <GenerateCompatibilitySuppressionFile>true</GenerateCompatibilitySuppressionFile>  
+  </PropertyGroup>
+
+</Project>
+```
+
+If `GenerateCompatibilitySuppressionFile` is set to `true`, all validation errors that occur during pack will be added as a suppression to the generated `CompatibilitySuppressions.xml` file.
+
+This file could be checked into source control to document and review the breaking changes made in a PR and the upcoming release.
```

Comment:
First of all, really great additions so far. Thank you for that.

The `GenerateCompatibilitySuppressionFile` property is intended to be passed in once, to generate or update the suppression file. It's not meant to be a property that you permanently set in your project file and then is true for all continuous builds. If you do so, you would accept other breaking as well by automatically getting them added to the suppression file.

Instead, the common case (or better to say, the case that we designed this for) is to set the `GenerateCompatibilitySuppressionFile=true` property just once. When working on the CLI you would do that via the following command: `dotnet pack /p:GenerateCompatibilitySuppressionFile=true`. Inside Visual Studio there isn't a great way to do this without creating distinct launch profiles (same for vscode). So you could set the property in the project file, then invoke `Pack` and then after the suppression file is generated / updated, remove the property from the project file.